docs/core_docs/docs/integrations/storage/in_memory.mdx

```diff
@@ -0,0 +1,13 @@
+# In Memory Store
```

Comment:
Looks like in Python this is under `integrations/stores`:

https://python.langchain.com/docs/integrations/stores/in_memory